nfd/node-feature-rules-openshift.yaml

```diff
@@ -16,6 +16,9 @@ spec:
           matchExpressions:
             vendor: {op: In, value: ["8086"]}
             class: {op: In, value: ["0300", "0380"]}
+        - feature: kernel.loadedmodule
```

Comment:
Lines 19-21 need to be omitted since we load out-of-tree i915 kernel module. Upstream relies on in-tree kmod.

Reply:
updated thanks